AlphaWallet/TokenScriptClient/Models/AssetImplicitAttributes.swift

```diff
@@ -3,16 +3,16 @@
 import Foundation
 
 enum AssetImplicitAttributes: String, CaseIterable {
-    case name
```

Comment:
This shouldn't be changed. It's the "name" of a token.

Reply:
@hboon sussed